expo/tests/scheduler/test_add_event.py

```diff
@@ -1,100 +1,331 @@
+                      args=["GenericEvent", self.eventitem.eventitem_id])
+        response = self.client.get(url)
+        nt.assert_equal(response.status_code, 403)
+
+    def test_good_user_get_bad_event(self):
+        login_as(self.privileged_profile, self)
+        url = reverse(self.view_name,
+                      urlconf="scheduler.urls",
+                      args=["GenericEvent", self.eventitem.eventitem_id+1])
+        response = self.client.get(url, follow=True)
+        nt.assert_equal(response.status_code, 404)
+
+    def test_good_user_get_success(self):
+        login_as(self.privileged_profile, self)
+        url = reverse(self.view_name,
+                      urlconf="scheduler.urls",
+                      args=["GenericEvent", self.eventitem.eventitem_id])
+        response = self.client.get(url)
+        nt.assert_equal(response.status_code, 200)
+        nt.assert_in(self.eventitem.title,
+                     response.content)
+        nt.assert_in(self.eventitem.description,
+                     response.content)
+        nt.assert_in(str(self.eventitem.duration),
+                     response.content)
+
+    def test_good_user_get_class(self):
+        login_as(self.privileged_profile, self)
+        klass = ClassFactory()
+        url = reverse(self.view_name,
+                      urlconf="scheduler.urls",
+                      args=["Class", klass.eventitem_id])
+        response = self.client.get(url)
+        nt.assert_equal(response.status_code, 200)
+        nt.assert_in(klass.title,
+                     response.content)
+        nt.assert_in(klass.description,
+                     response.content)
+        nt.assert_in("1:00",
+                     response.content)
+        nt.assert_in(str(klass.teacher),
+                     response.content)
+
+    def test_good_user_minimal_post(self):
+        Conference.objects.all().delete()
+        Room.objects.all().delete()
+        context = ClassContext()
+        login_as(self.privileged_profile, self)
+        url = reverse(self.view_name,
+                      urlconf="scheduler.urls",
+                      args=["Class", context.bid.eventitem_id])
+        response = self.client.post(
+            url,
+            data={'event-day': context.days[0].pk,
+                  'event-time': "12:00:00",
+                  'event-location': context.room.pk,
+                  'event-max_volunteer': 3,
+                  'event-title': 'New Title',
+                  'event-description': 'New Description',
+                  },
+            follow=True)
+
+        assert_redirects(response, reverse('event_schedule',
+                                           urlconf='scheduler.urls',
+                                           args=["Class"]))
+
+        nt.assert_not_in('<ul class="errorlist">', response.content)
+        nt.assert_in('Events Information', response.content)
+        sessions = Event.objects.filter(eventitem=context.bid, max_volunteer=3)
```

Comment:
I prefer to make this a comparison to the initial state of the object. That way, if someone decides to change the behavior of the context you're using here, it's not going to break this test. 
Also, `sessions = context.bid.scheduler_events.filter(max_volunteer=3)` is a little nicer way to say the same thing, and obviates the need to import Event
It would be nice also to see this generalized into a function, but that's not a blocker.